CHANGES.md

```diff
@@ -85,6 +85,7 @@ To be released.
  -  `BlockSet<T>[int]` changed so as not to validate a block.  [[#231]]
  -  Improved read performance of `Block<T>.Hash` and `Transaction<T>.Id`. [[#228],
     [#234]]
+ -  `Swarm.StartAsync()` doesn't call `Swarm.StopAsync()` anymore.
```

Comment:
So users need some action due to this change?

Reply:
If there is a code that relied on this process (without explicitly calling `StopAsync()`), they should now explicitly call `StopAsync()`.